### PR TITLE
Prison wing now gives a negative moodlet

### PR DESCRIPTION
**Pull request by TeomanTheGreat**
## About The Pull Request
I don't know about you but if I was trapped in a literal prison I would be pretty upset.

## Why It's Good For The Game
Adds a bit more flavor to being in perma. I think it makes sense.

## Changelog
:cl: Phil Smith
add: There is now a negative moodlet to being in perma
/:cl:

### CODE REVIEW

**Comment by TerryTehCoder** [comment]
Maybe Security Staff should be exempt from this. It'd be kind of weird to go in to Perma to drop off or catch a prisoner and get a moodlet about being Trapped. 

I don't think general staff go into Perma enough for anything past security to be a concern.

**Comment by TeomanTheGreat** [comment]
> 
> 
> Maybe Security Staff should be exempt from this. It'd be kind of weird to go in to Perma to drop off or catch a prisoner and get a moodlet about being Trapped.
> 
> I don't think general staff go into Perma enough for anything past security to be a concern.

Sec officers (and everyone else) already get a mood debuff from being in the normal cell areas and the hall outside of it too so this isn't much of a stretch.

File: code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -972,6 +972,8 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/security/prison
 	name = "Prison Wing"
 	icon_state = "sec_prison"
+	mood_bonus = -2 // less than normal brig moodlet due to the perma brig being more spacious
+	mood_message = "<span class='warning'>Im trapped here with little hope of escape!\n</span>"
```
Comment (TerryTehCoder) [inline code/game/area/Space_Station_13_areas.dm]:
```suggestion
	mood_message = "<span class='warning'>I'm trapped here with little hope of escape!\n</span>"
```
Grammar is Good.

Comment (TeomanTheGreat) [reply code/game/area/Space_Station_13_areas.dm]: Done

**Comment by TeomanTheGreat** [comment]
Oh cool since I did commit suggestion on your comment now your icon is down in the commit too. Neat.

File: code/game/area/Space_Station_13_areas.dm [hunk 2]
```diff
@@ -972,6 +972,8 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/security/prison
 	name = "Prison Wing"
 	icon_state = "sec_prison"
+	mood_bonus = -2 // less than normal brig moodlet due to the perma brig being more spacious
```
Comment (Crossedfall) [inline code/game/area/Space_Station_13_areas.dm]: You can leave the brig though, eventually. You're never leaving perma, even the message defines it as such. Shouldn't the debuf be even higher?

Comment (TeomanTheGreat) [reply code/game/area/Space_Station_13_areas.dm]: Fair enough ill fix this when i get back home

**Review by Archanial — APPROVED** [review]

**Pull request merged**